docs/data-sources/cassandra_cluster_v2_instance.md

```diff
@@ -237,6 +240,22 @@ Settings to determine how resize requests will be performed for the cluster.<br>
 *___notify_support_contacts___*<br>
 <ins>Type</ins>: boolean, read-only<br>
 <br>Setting this property to `true` will notify the Instaclustr Account's designated support contacts on resize completion.<br><br>
+<a id="nested--debezium"></a>
+## Nested schema for `debezium`
+Adds the specified version of Debezium Connector Cassandra to the Cassandra cluster, as well as the Kafka Cluster Data Centre ID, Kafka VPC Type, and Kafka Topic Prefix.<br>
+### Read-only attributes
+*___kafka_topic_prefix___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Kafka Topic Prefix<br><br>
+*___kafka_cdc_id___*<br>
+<ins>Type</ins>: string (uuid), read-only<br>
+<br>Kafka Cluster Data Centre Id<br><br>
+*___version___*<br>
+<ins>Type</ins>: string, read-only<br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Debezium Connector Cassandra to the Cassandra cluster. --AVAILABLE_BUNDLE_VERSIONS_MARKER_V2_DEBEZIUM--<br><br>
```

Comment:
`--AVAILABLE_BUNDLE_VERSIONS_MARKER_V2_DEBEZIUM--`

this is unexpected, can you test local API doc to see if versions are listed properly?
https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/1074036743/Local+and+Preproduction+Testing+API+Documentation

Reply:
Hi Long,

Thanks for this, it should be fixed now.

Reply:
nice,
just noticed a conflict in Makefile 
except that, everything is good